pages/en/developers/sandbox-node.mdx

```diff
@@ -24,7 +24,8 @@ docker run \
   -d \
   --name mina \
   -e RUN_DEMO=true \
-  minaprotocol/mina-daemon:1.2.0-fe51f1e-devnet
+  -e CODA_PRIVKEY_PASS='' \
```

Comment:
i think this should be `MINA_PRIVKEY_PASS` (CODA is deprecated)

Reply:
You are correct, I will update it to "MINA_PRIVKEY_PASS". I have tested that using the mina naming works.

Reply:
Added a new commit to update as requested.